api/apps/geoprocessing/test/integration/planning-unit-inclusion/planning-unit-inclusion.e2e-spec.ts

```diff
@@ -0,0 +1,62 @@
+import { INestApplication } from '@nestjs/common';
+import { PromiseType } from 'utility-types';
+import { createWorld } from './world';
+import { bootstrapApplication, delay } from '../../utils';
+
+import { ScenarioPlanningUnitsInclusionProcessor } from '@marxan-geoprocessing/modules/scenario-planning-units-inclusion/scenario-planning-units-inclusion-processor';
```

Comment:
prettier didn't run?

Reply:
it did 🤷

Reply:
removed unused import tho https://github.com/Vizzuality/marxan-cloud/pull/258/commits/4ac31134f1a72bfa1f499577fd9f2aa2c1db2f6a